Nexpo/Repositories/TicketRepository.cs

```diff
@@ -54,6 +55,12 @@ public async Task<IEnumerable<Ticket>> GetAllForEvent(int eventId)
                                             .ToListAsync();
         }
 
+        public async Task<EventType> GetEventType(int ticketId)
+        {
+            return await _context.Tickets.Include(t => t.Event).Where(t => t.Id == ticketId).Select(t => t.Event.Type).FirstOrDefaultAsync();
```

Comment:
I am uncertain on how this will work in practice. From what I can tell, when doing `Select` on an enum, the `FirstOrDefaultAsync` will fall back on the first value of the enum as default, i.e. we return a `CompanyEvent` on invalid requests. Adding a default "fallback" or "failure" entry to the enum and syncing this with the corresponding frontend calls might resolve this.

Assuming this is only called with valid `ticketId` arguments it's probably ok to ignore this, but for testing/debugging purposes it might be nice to keep the errors verbose